sql/updates/reverse-dev.sql

```diff
@@ -105,3 +105,9 @@ ANALYZE _timescaledb_catalog.continuous_agg;
 
 DROP PROCEDURE timescaledb_experimental.move_chunk(REGCLASS, NAME, NAME, NAME);
 DROP PROCEDURE timescaledb_experimental.copy_chunk(REGCLASS, NAME, NAME, NAME);
+
+ALTER TABLE _timescaledb_catalog.chunk_copy_operation
+	DROP COLUMN compress_chunk_name;
```

Comment:
dropping a column needs  to completely rewrite the table otherwise you create a hazard for future changes to that table.

Reply:
Can you elaborate this one please?

Reply:
You must not use DROP COLUMN in a upgrade/downgrade script as DROP COLUMN only marks the column dropped in the postgres catalog but leaves metadata behind resulting in a later ADD COLUMN behaving differently between fresh install/updated/downgraded install as the assigned attnum will be different. To get completely rid of the column you need to completely rebuild the table. copy current data into temp table, drop current table, recreate table with correct schema, there should be many examples of this in previous upgrade/downgrade scripts.